src/Aggregations/MultiTermsAggregation.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Spatie\ElasticsearchQueryBuilder\Aggregations;
+
+use Spatie\ElasticsearchQueryBuilder\AggregationCollection;
+use Spatie\ElasticsearchQueryBuilder\Aggregations\Concerns\WithAggregations;
+use Spatie\ElasticsearchQueryBuilder\Aggregations\Concerns\WithMissing;
+
+class MultiTermsAggregation extends Aggregation
+{
+    use WithMissing;
+    use WithAggregations;
+
+    /** @var string[] */
+    protected array $fields;
+
+    protected ?int $size = null;
+
+    protected ?array $order = null;
+
+    /**
+     * @param string[] $fields
+     */
+    public static function create(string $name, array $fields): self
+    {
+        return new self($name, $fields);
+    }
+
+    /**
+     * @param string[] $fields
+     */
+    public function __construct(string $name, array $fields)
+    {
+        $this->name = $name;
+        $this->fields = $fields;
+        $this->aggregations = new AggregationCollection();
+    }
+
+    public function size(int $size): self
+    {
+        $this->size = $size;
+
+        return $this;
+    }
+
+    public function order(array $order): self
```

Comment:
I think it would make sense to add support for the `Sorting` interface that is included in this package.

An example of that would be:

```suggestion
    /**
     * @param Sorting|Sorting[]|null $sort
     */
    public function order(array $order): self
    {
        $this->sort = is_array($sort) ? $sort : [$sort];
```

And then down below, an `array_map` like this:


```php
        if ($this->sort) {
            $parameters['sort'] = array_map(fn (Sorting $sort) => $sort->toArray(), $this->sort);
        }
```

Reply:
I will add support for both order AND size, which I should have included from the start, thanks!